app/src/main/java/org/jellyfin/mobile/player/deviceprofile/DeviceProfileBuilder.kt

```diff
@@ -299,7 +307,9 @@ class DeviceProfileBuilder {
         private val FORCED_AUDIO_CODECS = arrayOf(*PCM_CODECS, "alac", "aac", "ac3", "eac3", "dts", "mlp", "truehd")
 
         private val EXO_EMBEDDED_SUBTITLES = arrayOf("srt", "subrip", "ttml")
+        private val EXO_EMBEDDED_SUBTITLES_ASS = arrayOf("srt", "subrip", "ttml", "ssa", "ass")
```

Comment:
Alternatively, you could do just one constant that only contains the ssa/ass formats (`private val SUBTITLES_SSA = arrayOf("ssa", "ass")`), and append the array to the other one on callsite, like `EXO_EMBEDDED_SUBTITLES + SSA_SUBTITLES`.